applications/StructuralMechanicsApplication/python_scripts/convergence_criteria_factory.py

```diff
@@ -31,6 +31,11 @@ def __init__(self, convergence_criterion_parameters):
             if convergence_criterion_parameters["volumetric_strain_dofs"].GetBool():
                 volumetric_strain_dofs = True
 
+        strain_vector_dofs = False
+        if convergence_criterion_parameters.Has("strain_vector_dofs"):
+            if convergence_criterion_parameters["strain_vector_dofs"].GetBool():
+                strain_vector_dofs = True
```

Comment:
```suggestion
        strain_dofs = False
        if convergence_criterion_parameters.Has("strain_dofs"):
            if convergence_criterion_parameters["strain_dofs"].GetBool():
                strain_dofs = True
```

I'd call it `strain_dofs` for the sake of consistency (it is less verbose as well).